src/ontodia/editor/editorController.tsx

```diff
@@ -513,6 +517,67 @@ export class EditorController {
         }
 
         this.setSelection(placedElements);
+
+        this.source.trigger('addElements', { elements: placedElements });
+    }
+
+    onAddElementsInConnectionMenu(
```

Comment:
Adding elements operation was extracted from connection menu to the editorController, to make it possible to fire event on adding elements.